Makefile

```diff
@@ -38,7 +38,7 @@ build_python:: schema
 		cp ../README.md . && \
 		python3 setup.py clean --all 2>/dev/null && \
 		rm -rf ./bin/ ../python.bin/ && cp -R . ../python.bin && mv ../python.bin ./bin && \
-		sed -i.bak -e "s/\$${VERSION}/$(PYPI_VERSION)/g" -e "s/\$${PLUGIN_VERSION}/$(VERSION)/g" ./bin/setup.py && \
+		sed -i.bak -e 's/^VERSION = .*/VERSION = "$(PYPI_VERSION)"/g' -e 's/^PLUGIN_VERSION = .*/PLUGIN_VERSION = "$(VERSION)"/g' ./bin/setup.py && \
```

Comment:
Curious why this change was needed?

Reply:
Justin made a similar change a few weeks ago in `pulumi-kubernetes` - check the rationale here: https://github.com/pulumi/pulumi-kubernetes/pull/1664